concourse/pipelines/artifact-releaser-test.jsonnet

```diff
@@ -390,8 +390,8 @@ local pkggroup = {
     '.0@1.goo',
     '@1.goo',
     '@1.goo',
-    '.0@1.goo',
     '@1.goo',
+    '.0@1.goo',
```

Comment:
I'm curious to know/understand this ordering change impact.

Reply:
So every index of file endings is matched with the same index in (in this case) gcs package names. I mismatched them, causing some errors in the testing pipeline.